microsoft-outlook/latest/install/index.md

```diff
@@ -58,7 +58,22 @@ You can use one of the following Outlook releases:
 
 ### Alfresco requirements
 
-* Alfresco Content Services 6.2.2 or later. See [Supported Platforms]({% link microsoft-outlook/latest/support/index.md %}) for more information.
+* Alfresco Content Services 6.2.2 or later.
+
+#### Alfresco Search and Insight Engine 2.0 and above
+
+If you're using Alfresco Search and Insight Engine 2.0 and above in combination with Outlook Integration 2.8.1 and above, a property needs to be added to the `shared.properties` file for SOLR. See the [Alfresco indexing recommendations]({% link insight-engine/latest/config/indexing.md %}#cross-locale) to locate this file.
```

Comment:
replace "Alfresco Search and Insight Engine 2.0 and above" with
"Alfresco Search Services 2.0 and above"